### PR TITLE
kubeadm cleanup: master -> control-plane (cont.5)

### PR DESCRIPTION
**Pull request by vanduc95**
This PR continuation of [#73987](https://github.com/kubernetes/kubernetes/pull/73987)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/assign @neolit123

### CODE REVIEW

File: cmd/kubeadm/app/util/config/initconfiguration_test.go [hunk 1]
```diff
@@ -34,16 +34,16 @@ import (
...
-	masterV1alpha3YAMLNonLinux  = "testdata/conversion/master/v1alpha3_non_linux.yaml"
-	masterV1beta1YAML           = "testdata/conversion/master/v1beta1.yaml"
-	masterV1beta1YAMLNonLinux   = "testdata/conversion/master/v1beta1_non_linux.yaml"
-	masterInternalYAML          = "testdata/conversion/master/internal.yaml"
-	masterInternalYAMLNonLinux  = "testdata/conversion/master/internal_non_linux.yaml"
-	masterIncompleteYAML        = "testdata/defaulting/master/incomplete.yaml"
-	masterDefaultedYAML         = "testdata/defaulting/master/defaulted.yaml"
-	masterDefaultedYAMLNonLinux = "testdata/defaulting/master/defaulted_non_linux.yaml"
-	masterInvalidYAML           = "testdata/validation/invalid_mastercfg.yaml"
+	controlPlaneV1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
```
Comment (neolit123) [inline cmd/kubeadm/app/util/config/initconfiguration_test.go]: we need to change `master` -> `controlplane` in the filenames and folder too if we are changing the constant names.

**Review by neolit123 — COMMENTED** [review]
thanks a lot @vanduc95 !
added one comment.

File: cmd/kubeadm/app/util/config/initconfiguration_test.go (hunk 1 above)
Comment (vanduc95) [reply cmd/kubeadm/app/util/config/initconfiguration_test.go]: yes, you were right ! 
I have just updated.

Comment (neolit123) [reply cmd/kubeadm/app/util/config/initconfiguration_test.go]: awesome! thanks.
/ok-to-test

**Comment by neolit123** [comment]
/approve
@kubernetes/sig-cluster-lifecycle-pr-reviews

**Review by rosti — APPROVED** [review]
Thanks @vanduc95 !
/lgtm

**Pull request merged**